libs/blocks/global-navigation/global-navigation.js

```diff
@@ -63,25 +63,26 @@ const decorateSignIn = async ({ rawElem, decoratedElem }) => {
   let signInElem;
 
   if (!dropdownElem) {
-    signInElem = toFragment`<a href="#" daa-ll="${signInLabel}" class="feds-signIn">${signInLabel}</a>`;
+    signInElem = toFragment`<button qa="gnav-ims-sign-in" daa-ll="${signInLabel}" class="feds-signIn">${signInLabel}</button>`;
 
     signInElem.addEventListener('click', (e) => {
       e.preventDefault();
       signIn();
     });
   } else {
-    signInElem = toFragment`<a href="#" daa-ll="${signInLabel}" class="feds-signIn" role="button" aria-expanded="false" aria-haspopup="true">${signInLabel}</a>`;
+    signInElem = toFragment`<button qa="gnav-ims-sign-in" daa-ll="${signInLabel}" class="feds-signIn" aria-expanded="false" aria-haspopup="true">${signInLabel}</button>`;
 
     signInElem.addEventListener('click', (e) => trigger({ element: signInElem, event: e }));
     signInElem.addEventListener('keydown', (e) => e.code === 'Escape' && closeAllDropdowns());
     dropdownElem.addEventListener('keydown', (e) => e.code === 'Escape' && closeAllDropdowns());
 
     dropdownElem.classList.add('feds-signIn-dropdown');
 
-    const dropdownSignIn = dropdownElem.querySelector('[href$="?sign-in=true"]');
-
-    if (dropdownSignIn) {
-      dropdownSignIn.addEventListener('click', (e) => {
+    const dropdownSignInAnchor = dropdownElem.querySelector('[href$="?sign-in=true"]');
+    if (dropdownSignInAnchor) {
+      const dropdownSignInButton = toFragment`<button qa="gnav-ims-sign-in">${dropdownSignInAnchor.innerText}</button>`;
```

Comment:
I'd suggest the use of `textContent` instead of `innerText`. The latter needs layout information, the former is more performant and standartdised.